rules.txt

```diff
@@ -1,12 +1,395 @@
+#    Compliant with the site-wide reddit rules.
+#Comment Rules:
+#  All comments MUST be:
+#    Compliant with the site-wide reddit rules.
+##################
+# Required Link Flair names:
+## Text: "Enlightenment Developer" Flair:enlightmentdeveloper
+## Text: "KDE Developer" Flair: kdedeveloper
+## Text: "Kernel Developer" Flair: coredeveloper
+## Text: "Gnome Developer" Flair: gnomedeveloper
+## Text: "Distro News" Flair: distronews
+## Text: "Popular Application" Flair: popularapplication
+## Text: "AlternativeOS" Flair: alternativeos
+## Text: "Microsoft" Flair: microsoft
+## Text: "Mobile OS" Flair: mobilelinuxbased
+## Text: "Removed" Flair: removed
+##################
+---
+##----------------------------
+##    Affiliate Links Rules
+##----------------------------
+    standard: amazon affiliate links
+    action: filter
+    action_reason: Amazon referral
+---
+##----------------------------
+##    Click-bait Rules
+##----------------------------
+    title+media_title (regex): ["character are you","\\d{1,2} things only","before you die","is this the","you probably d(id|o)n't","are the most","in your life","\\d{1,2} things that","things you probably","reasons you should","probably d(id|o)n't know","\\d{1,2} most important","\\d{1,2} (signs|reasons) (you('re)?|why)","\\d{1,2} most important","d(id|o)n't know about", "(photos|pictures|images) that prove","blow your mind","game of thrones","things that (actually |really )?happen(ed)?","in real life","will blow your","\\d{1,2} dogs who","\\d{1,2} things you", "you should be", "things you d(id|o)n't"]
+    action: report
+    action_reason: "Possible Clickbait - {{match-1}}"
+---
+##----------------------------
+##    Disguised Links Rules, from automoderator library
+##----------------------------
+    moderators_exempt: false
+    body (regex): "(\\[(?P<text>(http|www)\\S+)\\]\\((?!(?P=text))(http|www)\\S+\\))"
+    action: remove
+    action_reason: "Disguised link"
+    modmail: The above {{kind}} by /u/{{author}} was removed because it contained a disguised link. Please investigate immediately as the user was likely trying to circumvent rules.
+---
+##----------------------------
+## Language/Profanity (mostly from automoderator example) - Severe - remove, message user, and modmail
+##----------------------------
+    moderators_exempt: true
+    title+body (regex): ["cock[sz]?([ -]?suck(er[sz]?|ing)?)?","cum(ming|[ -]shot[sz]?)?", "n[ioe0@][g]{1,}[aioe0@]{1,}r[sz]?", "f[aioe0@]{1,}[g]{1,}[aioe03@]{1,}t[sz]?", "k[i1]ke[sz]?"]
+    action: remove
+    action_reason: "Severe Profanity"
+    message: |
+        Your [{{kind}}]({{permalink}}) in /r/{{subreddit}} was automatically removed for use of extreme profanity.
+
+        /r/{{subreddit}} is for productive discussion, so please watch your language.
+    modmail: The above post by /u/{{author}}, contained profanity not becoming of the /r/Linux community. The comment was removed. Please review users account for further action.
```

Comment:
Should posts about TempleOS be excluded, because these words are often needed to describe the actions of Terry Davis?

Reply:
Of course, could just leave it manually done

Reply:
I was writing rules about tagging alternate OS's and thought it'd be good to put a reminder about TempleOS since that comes around here and there. If I could exempt the comment section for the few TempleOS posts I would but I don't believe that's possible. The rules and reminder exist to note that just because Terry says it doesn't mean a comment about it will be allowed. The posts are rare enough that manual approval can be done if mods want to.